### DISCUSSION TITLE
Very simple question, can I tell Sparkle to manually update, and simply not use the appcast system?

### DISCUSSION THREAD

**Discussion by smhk** [sparkle-project/Sparkle#2242] [Q&A]
Can I tell Sparkle to manually update (giving it a url), and simply not use the appcast system?

TY!

(Could not find the question here and thought it might be canonical, TY)

**Comment by zorgiepoo** [comment]
No. (And anyway at the bare minimum Sparkle needs to know if the app is already up to date or not).
